extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -472,6 +472,9 @@ export class AmpSlideScroll extends BaseSlides {
    * @param {number} currentScrollLeft scrollLeft value of the slides container.
    */
   updateOnScroll_(currentScrollLeft) {
+    if (currentScrollLeft == NaN || this.slideIndex_ == null) {
```

Comment:
You can use `!isFiniteNumber(currentScrollLeft)` from `types.s` instead of `NaN` check